prov/psm/src/psmx_init.c

```diff
@@ -558,6 +558,7 @@ static int psmx_getinfo(uint32_t version, const char *node, const char *service,
 	psmx_info->domain_attr->mr_iov_limit = 65535;
 	psmx_info->domain_attr->caps = PSMX_DOM_CAPS;
 	psmx_info->domain_attr->mode = 0;
+	psmx_info->domain_attr->mr_cnt = 10000;
```

Comment:
To be clear, the fields are unsigned, so the value is better represented as SIZE_MAX, not -1. I would interpret such a value to mean that the provider limit is tied to memory constraints.  This applies to any of the size_t attribute fields.